addons/udes_stock/models/res_users.py

```diff
@@ -1,10 +1,41 @@
 from odoo import api, fields, models, SUPERUSER_ID, _
+from odoo.addons.udes_common.models.fields import PreciseDatetime
 from odoo.exceptions import ValidationError
+import logging
+
+_logger = logging.getLogger(__name__)
+
+NON_ASSIGNABLE_STATES = ("draft", "done", "cancel")
 
 
 class ResUser(models.Model):
     _inherit = "res.users"
 
+    u_picking_id = fields.Many2one(
+        "stock.picking",
+        string="Assigned Picking",
+        help="The picking a user is currently assigned to",
+        readonly=True,
+    )
+    u_picking_assigned_precise_time = PreciseDatetime(
+        string="User Date Started (microseconds)",
+        help="Precise date with microseconds when the user got assigned to the picking.",
```

Comment:
It makes more sense to use "time" or "datetime" here rather than "date".

Reply:
I think the `string` value needs to the same change?

Reply:
Changed